app/models/hud_reports/question_base.rb

```diff
@@ -7,6 +7,7 @@
 module HudReports
   class QuestionBase
     include NotifierConfig
+    include HudApr::CellDetailsConcern
```

Comment:
Ah, I missed this.  This can't go here because `HudReports::QuestionBase` serves more than just the Apr (and related reports).  This will need to be mixed into each question, or an APR specific question base class.

Reply:
Also, you can't depend on the driver being active, so this can't be included here unless the concern is also outside of the driver... (which we don't want)

Reply:
✅ We don't need it here with the latest strategy